prometheus-exporters/vrops-exporter/alerts/vccluster.alerts

```diff
@@ -0,0 +1,34 @@
+groups:
+- name: vccluster.alerts
+  rules:
+  - alert: vCenterDRS_DisabledWithRegisteredVMs
+    expr: >
+      max_over_time(vrops_cluster_properties{propkey="configuration_drsconfig_enabled: false"}[10m]) 
+      and on (vccluster) max_over_time(vrops_cluster_stats{statkey="summary_total_number_vms"}[10m]) > 0
+    for: 20m
+    labels:
+      severity: info
+      tier: vmware
+      service: compute
+      context: "vrops-exporter"
+      meta: "Cluster Compute Resource {{ $labels.vccluster }} has disabled DRS with registered VMs  ({{ $labels.datacenter }})"
+      playbook: 
+    annotations:
+      description: "Cluster Compute Resource {{ $labels.vccluster }} has disabled DRS with registered VMs ({{ $labels.datacenter }})"
+      summary: "Cluster Compute Resource {{ $labels.vccluster }} has disabled DRS with registered VMs ({{ $labels.datacenter }})"
+  - alert: vCenterDRS_fullyAutomated_DisabledWithRegisteredVMs
+    expr: >
+      max_over_time(vrops_cluster_properties{propkey=~"configuration_drsconfig_enabled.*"}[10m]) == 1 
```

Comment:
Name is a bit confusing and so the description. Took me a bit to understand what it is about.

Maybe like this:
VCenterAutomatedVMsGotDisabledDRS

Please adopt the description lines accordingly